packages/my-near-wallet/README.md

```diff
@@ -35,15 +35,15 @@ const selector = await setupWalletSelector({
 ## Options
 
 - `walletUrl` (`string?`): Wallet URL used to redirect when signing transactions. This parameter is required for custom network configuration.
-- `iconUrl`: (`string?`): Image URL for the icon shown in the modal. This can also be a relative path or base64 encoded image. Defaults to `./assets/my-near-wallet-icon.png`.
+- `iconUrl`: (`string?`): Image URL for the icon shown in the modal. This can also be a relative path or base64 encoded image. Defaults to `./assets/my-near-wallet-icon.svg`.
 
 ## Assets
 
 Assets such as icons can be found in the `/assets` directory of the package. Below is an example using Webpack:
 
 ```ts
 import { setupMyNearWallet } from "@near-wallet-selector/my-near-wallet";
-import myNearWalletIconUrl from "@near-wallet-selector/my-near-wallet/assets/my-near-wallet-icon.png";
+import myNearWalletIconUrl from "@near-wallet-selector/my-near-wallet/assets/my-near-wallet-icon.svg";
```

Comment:
Hey @followdarko thanks for raising this PR.

Yesterday we have published a new version [v5.0.0](https://www.npmjs.com/package/@near-wallet-selector/core) of the wallet selector, and for now we've made a decision to use the original (official) icons of each wallet.

You can have a look at it here: https://near.github.io/wallet-selector/